scripts/wrappers/start.sh

```diff
@@ -45,6 +61,8 @@ function start_opensearch () {
 
     warn_if_missing_perm "process-control"
 
+    configure_qat
```

Comment:
Just making sure I understand correctly - at this point the `start.sh` script is being run as a daemon by `root`, correct? Otherwise the `setfacl` would fail.